datagen/data/departments.json

```diff
@@ -26762,7 +26762,7 @@
         "description": "An introduction to the fundamental data structures and algorithms underlying many parts of Bioinformatics.  Standard data structures for efficient indexing and sequence search will be covered, including the suffix array and  the FM-index, as will alignment-free methods for sequence comparison. This course will also introduce the  fundamental algorithms in computational phylogenomics and biological network analysis. Finally, bioinformatics oriented applications of classic unsupervised learning algorithms (e.g., clustering and dimensionality reduction) and database techniques (e.g., sorting, selection, joining) will be examined. The focus will be both on formal  understanding of computational efficiency as well as the practical applications of these concepts.",
         "sections": [
           {
-            "sec_code": "1",
+            "sec_code": "S1",
```

Comment:
Issue: this is not correct. See: [BIOI607](https://app.testudo.umd.edu/soc/search?courseId=BIOI607&sectionId=&termId=202401&_openSectionsOnly=on&creditCompare=%3E%3D&credits=0.0&courseLevelFilter=ALL&instructor=&_facetoface=on&_blended=on&_online=on&courseStartCompare=&courseStartHour=&courseStartMin=&courseStartAM=&courseEndHour=&courseEndMin=&courseEndAM=&teachingCenter=ALL&_classDay1=on&_classDay2=on&_classDay3=on&_classDay4=on&_classDay5=on); the section code should be PCS1, not S1.

Reply:
These sections are now included.